src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -77,12 +80,13 @@ namespace pluginhost {
                                         std::string const &name,
                                         OSVR_PluginRegContext ctx,
                                         bool shouldRethrow = false) {
-        OSVR_DEV_VERBOSE("Trying to load a plugin with the name " << name);
+        auto log = util::log::make_logger("PluginHost");
+        log->debug() << "Trying to load a plugin with the name " << name;
```

Comment:
I should think we should be able to configure the sinks with different levels - if not, then I'm not sure why we have a logging system this sophisticated....